### PR TITLE
Routing, File and History Views

### PR DESCRIPTION
**Pull request by drewatk**

### CODE REVIEW

File: app/containers/EditorPage.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { connect } from "react-redux";
+import ProseEditor from "app/components/ProseEditor";
+import TitleBar from "app/components/TitleBar";
+import FileList from "app/components/FileList";
+import History from "app/components/History";
+
+const EditorPage = props => {
+  const { showHistory, showFileList } = props;
+  return (
```
Comment (drewatk) [inline app/containers/EditorPage.js]: @marsha88 this is where I'm putting the layout of the page

File: app/actions/view.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export const TOGGLE_SHOW_HISTORY = "TOGGLE_SHOW_HISTORY";
+export const TOGGLE_SHOW_FILE_LIST = "TOGGLE_SHOW_FILE_LIST";
+
+export function toggleShowHistory() {
```
Comment (marsha88) [inline app/actions/view.js]: Any reason these two actions aren't just plain objects?

File: app/containers/Root.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react";
 import { Provider } from "react-redux";
-import { ConnectedRouter } from "react-router-redux";
+import { ConnectedRouter } from "connected-react-router";
```
Comment (marsha88) [inline app/containers/Root.js]: I swear a new routing system for react comes out every week lol.
Haven't heard of this one, but looks nice!

Comment (drewatk) [reply app/containers/Root.js]: react-router-redux got deprecated in favor of this so I figured might as well

File: app/actions/view.js (hunk 2 above)
Comment (drewatk) [reply app/actions/view.js]: Following the pattern of the boilerplate...

File: app/containers/Root.js (hunk 3 above)
Comment (drewatk) [reply app/containers/Root.js]: also not a new routing system, just redux bindings for react-router

File: app/actions/view.js (hunk 2 above)
Comment (marsha88) [reply app/actions/view.js]: ok that's fine. Usually actions aren't functions unless you need to modify the payload of the action dispatched during the dispatch call.

**Pull request merged**